modules/gumgumBidAdapter.js

```diff
@@ -310,10 +328,10 @@ function buildRequests(validBidRequests, bidderRequest) {
     for (const eid in eids) data[eid] = eids[eid];
 
     // ADJS-1024 & ADSS-1297
-    if (deepAccess(ortb2Imp, 'ext.data.pbadslot')) {
-      gpid = deepAccess(ortb2Imp, 'ext.data.pbadslot')
-    } else if (deepAccess(ortb2Imp, 'ext.data.adserver.name')) {
-      gpid = ortb2Imp.ext.data.adserver.adslot
+    if (gpid = deepAccess(ortb2Imp, 'ext.data.pbadslot')) {
+      data.gpid = gpid;
```

Comment:
you could do:

data.gpid = deepAccess(CONDITION 1) || deepAccess(CONDITION 2) 

This is assuming that only one of the conditions will be true at a time.

Reply:
yeah i think i was cleaning up old logic but didn't think out how to make it shorter. the line length would be pretty long in that case so i'll try it out and see